cisa-advisory/info.json

```diff
@@ -0,0 +1,407 @@
+                        "2016",
+                        "2015",
+                        "2014",
+                        "2013",
+                        "2012",
+                        "2011",
+                        "2010"
+                    ]
+                }
+            ],
+            "output_schema": {
+                "url": "",
+                "name": "",
+                "icsa_code": "",
+                "year": 0,
+                "vendor": "",
+                "product": "",
+                "cve_list": [],
+                "cwe_list": [],
+                "release_date": "",
+                "updated_date": "",
+                "cvss_severity": "",
+                "cvss_score": "",
+                "affected_product": "",
+                "company_headquaters_location": "",
+                "country_area_deployed": "",
+                "critical_infrastructure_sector": ""
+            }
+        },
+        {
+            "operation": "get_ics_medical_advisory_by_product",
+            "title": "Get ICS Medical Advisory By Product",
+            "annotation": "get_ics_medical_advisory_by_product",
+            "description": "Fetches ICS Medical Advisory on the basis of specified product name and version.",
+            "category": "investigation",
+            "is_config_required": true,
+            "visible": true,
+            "enabled": true,
+            "parameters": [
+                {
+                    "name": "product",
+                    "title": "Product",
+                    "type": "text",
+                    "editable": true,
+                    "visible": true,
+                    "required": true,
+                    "tooltip": "Specify the product to get ICS Advisory",
+                    "description": "Specify the product to get ICS Advisory"
+                },
+                {
+                    "name": "version",
+                    "title": "Version",
+                    "type": "text",
+                    "editable": true,
+                    "visible": true,
+                    "required": false,
+                    "tooltip": "Specify the version of product to get ICS Advisory",
+                    "description": "Specify the version of product to get ICS Advisory"
+                }
+            ],
+            "output_schema": {
+                "url": "",
+                "name": "",
+                "icsa_code": "",
+                "year": 0,
+                "vendor": "",
+                "product": "",
+                "cve_list": [],
+                "cwe_list": [],
+                "release_date": "",
+                "updated_date": "",
+                "cvss_severity": "",
+                "cvss_score": "",
+                "affected_product": "",
+                "company_headquaters_location": "",
+                "country_area_deployed": "",
+                "critical_infrastructure_sector": ""
+            }
+        }
+    ],
+    "category": "Utilities",
```

Comment:
PLease check the category I believe it should be investigation

Reply:
As there is no input to investigate, only we are fetching data from our prod repo